internal/trigger/transform/template/exec.go

```diff
@@ -58,5 +60,13 @@ func (t *Template) Execute(data map[string]interface{}) []byte {
 		}
 	}
 	stream.Flush()
-	return sb.Bytes()
+	bytes := sb.Bytes()
+	if t.contentType == "" {
+		if gjson.ValidBytes(bytes) {
```

Comment:
why here has a judgment? is there someplace to set contentType?

Reply:
the template will create a new instance if the user changes the subscription template, so the template instance can't be changed. the template instance property "contentType" only is set once at here and  can't be changed